Sources/TuistGraph/Models/SwiftPackageManagerDependencies.swift

```diff
@@ -10,4 +10,58 @@ public struct SwiftPackageManagerDependencies: Equatable {
     public init(_ packages: [Package]) {
         self.packages = packages
     }
+
+    /// Returns `Package.swift` representation.
+    ///
+    /// **NOTE** It is a temporary solution until Apple resolves: https://forums.swift.org/t/pitch-package-editor-commands/42224
+    public func stringValue() -> String {
+        """
+        // swift-tools-version:5.3
+
+        import PackageDescription
+
+        let package = Package(
+            name: "PackageName",
+            dependencies: [
+                \(packages.map { $0.stringValue + "," }.joined(separator: "\n        "))
+            ]
+        )
+        """
+    }
+}
+
+// MARK: - Package.stringValue()
+
+private extension Package {
```

Comment:
I think this method can be part of `Package.swift`. Moreover, I think `manifestValue` would be more accurate because it's the format of the package when used in a manifest file.